README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with StellarStation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [StellarStation](https://www.stellarstation.com/) API in GNURadio flowgraphs.
+
+## Installation
+
```

Comment:
Can we add target OS of this build? I guess it only work in Linux environment.

Reply:
I don't see a reason why it wouldn't work on any OS running GNURadio.